api/dbadapters/bigquery.ts

```diff
@@ -232,41 +232,57 @@ export class BigQueryDbAdapter implements IDbAdapter {
     return rows;
   }
 
-  public async schemas(database: string): Promise<string[]> {
-    const data = await this.getClient(database).getDatasets();
-    return data[0].map(dataset => dataset.id);
+  public async databases(): Promise<string[]> {
+    throw new Error(
+      "The Dataform BigQuery API adapter does not (yet) support multiple databases in the same connection"
```

Comment:
GH won't let me respond to the earlier comment chain (ugh), but in response: *all* GCP credentials are user-scoped. a service account is still a "user", it's just not a human.

i'm not going to block on adding `databases()` support for BQ connections, but it should be fairly simple, you should be able to issue the API call pretty easily right here.